public/ide/views/page/page.view.html

```diff
@@ -1 +1,127 @@
-<h1>page</h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <script
```

Comment:
common code should sit in index,html. So i guess it should be moved to index.html.